packages/core/tocco-ui/src/Select/StyledComponents.js

```diff
@@ -192,6 +193,9 @@ export const reactSelectStyles = outerTheme => {
     }),
     input: (base, state) => {
       let indicatorWidth = 50
+      if (state.selectProps.hasDocsTreeSearch) {
+        indicatorWidth += 25
+      }
       if (state.selectProps.hasAdvancedSearch) {
```

Comment:
Yes, this is because it should be possible to show advancedSearch and docsTreeSearch in the future.
![Screen Shot 2022-11-24 at 08 05 13](https://user-images.githubusercontent.com/6946713/203716421-a1b3a232-a307-449b-9b9c-d95d2cd86427.png)